test/functionalTest/cases/3458_legacy-forwarding-with-idpattern/legacy-forwarding-with-idpattern.test

```diff
@@ -0,0 +1,400 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I understand is out of the scope of issue #3458 but, what happens with update? Maybe it is works as it is now, without any additional change (as [it seems](https://github.com/telefonicaid/fiware-orion/pull/3459/files#r267441346) happended for forwarded queries).

Thus, I'd suggest to check with with a small new .test. Two possiblities:

1. It works. Great :). Add the .test to the PR and we are done.
2. It doeesn't work. Oh :(. Include in this PR a remark in the documentation about it. A new issue will be created for the uncovered functionality (i.e. "Update forward support with registrations with idPattern `".*"`")

Reply:
Sure, I can implement a small test

Reply:
Test performed and it doesn't work.
First error: empty entity id is not allowed.
The very next PR will take care of updates.

Reply:
Doc fixed in 1726bd308